modules/client/src/main/java/org/apache/ignite/client/ClientOperationType.java

```diff
@@ -193,5 +194,15 @@ public enum ClientOperationType {
     /**
      * Get all primary replicas mapping to cluster nodes ({@link PartitionManager#primaryReplicasAsync()}).
      */
-    PRIMARY_REPLICAS_GET
+    PRIMARY_REPLICAS_GET,
+
+    /**
+     * Get tables ({@link IgniteTables#tables()}) that returns qualified names.
+     */
+    TABLES_GET_QUALIFIED,
+
+    /**
+     * Get table ({@link IgniteTables#table(QualifiedName)}).
+     */
+    TABLE_GET_QUALIFIED,
```

Comment:
This enum exists for the purpose of retry policy implementation. It is public, unlike `ClientOp`.

I think we can map internal `TABLES_GET_QUALIFIED` to public `TABLES_GET`, because those are the same from the user point of view.